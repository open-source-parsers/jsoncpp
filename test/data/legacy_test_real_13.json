[-1e+9999, 1e+9999]
