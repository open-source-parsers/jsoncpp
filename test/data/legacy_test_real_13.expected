.=[]
.[0]=-inf
.[1]=inf
